lib/vrl/vrl/benches/vm.rs

```diff
@@ -0,0 +1,67 @@
+use std::collections::BTreeMap;
+
+use compiler::state;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use indoc::indoc;
+use vector_common::TimeZone;
+use vrl::{Runtime, Value};
+
+static SOURCE: [(&str, &str); 2] = [
```

Comment:
Instead of a tuple please make this a struct. The meanings of the fields are much more clear that way; although I can infer what they're supposed to be now that won't be the case at 3+ fields necessarily.